### PR TITLE
fix(zh_TW): update strings and added credits

### PR DESCRIPTION
**Pull request by olivertzeng**

### CODE REVIEW

File: zh-TW.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,13 +1,13 @@
 //
 //	Translation by Oliver Tzeng
-//
+//  Some fixes by Neo1102
```
Comment (L1ghtmann) [inline zh-TW.lproj/Localizable.strings:3]: Can you move this up with the other contributor

**Review by L1ghtmann — CHANGES_REQUESTED** [review]
Looks good, thanks! Just have one comment

**Comment by olivertzeng** [comment]
> Looks good, thanks! Just have one comment 

Done

**Review by L1ghtmann — APPROVED** [review]
Thanks again

**Pull request merged**